Meadowlark-js/services/meadowlark-fastify/src/Service.ts

```diff
@@ -119,9 +120,8 @@ export function buildService(): FastifyInstance {
     fastify.get(`/${stage}/metadata/descriptors/swagger.json`, swaggerForDescriptorsAPI);
     fastify.get(`/${stage}/metadata/data/v3/dependencies`, dependencies);
 
-    // OAuth handlers
+    // Old OAuth handlers
```

Comment:
🤨

Reply:
The old OAuthHandler in meadowlark-core is only 9 lines of code now.